### PR TITLE
Article for smithay v0.1

### PR DESCRIPTION
**Pull request by elinorbgr**

### CODE REVIEW

File: content/v0.1.markdown [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+better to just start over with a new protocol. The project has been going for a few years now and adoption
+is getting pretty good. Notably Gtk/Gnome and Qt/KDE are now almost fully compatible, and some linux
+distributions have started enabling it by default.
+
+In a few points, here are the main difference between Wayland and X11:
+
+- Wayland is much smaller and barebones. We could push the analogy to the point of saying that Wayland
+  is to X11 what Vulkan is to OpenGL. For example, a Wayland client is expected to fully handle the
+  drawing of its content, and just hand a buffer to the server via shared memory, while the X11 protocol
+  embeds a log of drawing primitives (though many recent X11 apps already do it that way anyway).
```
Comment (Drakulix) [inline content/v0.1.markdown]: "a log of"?

File: content/v0.1.markdown [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+of code is still needed for a compositor built on smithay to work, and there is not nearly enough
+space in this blog post to paste the code.
+
+You can also check out [Smithay's documentation on docs.rs](https://docs.rs/smithay). The crate-level
+docs are minimalistic, but we try to expand on all the details in the documentation of the individual
+modules.
+
+## What's next?
+
+For the 0.2 target, we plan to keep workin on the fundamentals. Some of the most important remaining features are:
```
Comment (Drakulix) [inline content/v0.1.markdown]: "(and possibly 0.3)" maybe?

**Review by Drakulix — APPROVED** [review]
I have just found a simple typo.

Content-wise this is all good. No misinformation to be spotted anywhere, and everything is really on point and reads very fluently.

I am just not sure about the goals for 0.2. I am not sure if we might be committing to too many features. I understand that this wants to be a list of essential features, that we are still missing, but it reads like we want to have all of them for 0.2.

I think a more fast-paced development approach would benefit the project and e.g. getting everything but XWayland to work, would already be a huge step, that would justify a release.

File: content/v0.1.markdown (hunk 1 above)
Comment (elinorbgr) [reply content/v0.1.markdown]: ahah, that should be "many" actually I think (I was thinking "a lot of")

File: content/v0.1.markdown (hunk 2 above)
Comment (elinorbgr) [reply content/v0.1.markdown]: Yeah, you're right, maybe I've been overloading the 0.2 milestone

**Pull request merged**